foundation/auth/Cargo.toml

```diff
@@ -11,7 +11,7 @@ description = "Google Cloud Platform server application authentication library."
 
 [dependencies]
 tracing = "0.1"
-reqwest = { version = "0.11", features = ["json"], default-features = false }
+reqwest = { version = "0.12.4", features = ["json"], default-features = false }
```

Comment:
`charset` feature is required to use `text_with_charset`  at  0.12.4 
https://docs.rs/reqwest/0.12.4/reqwest/struct.Response.html#method.text_with_charset

Reply:
I added this feature to `foundation/auth/Cargo.toml`.